libraries/provider_component_runit_supervisor_upstart.rb

```diff
@@ -4,11 +4,11 @@ class Chef
   class Provider
     class ComponentRunitSupervisor
       class Upstart < Chef::Provider::LWRPBase
-        provides :component_runit_supervisor, :platform_family => "debian"
         provides :component_runit_supervisor, :platform_family => "rhel" do |node|
           node['platform_version'].to_i == 6
         end
-        provides :component_runit_supervisor, :platform => %w[ amazon fedora ]
+        provides :component_runit_supervisor,
+          :platform => %w[ amazon fedora ubuntu ]
```

Comment:
No this just explicitly adds ubuntu where before it was being chose by process of elimination. I suppose I can make it a separate commit, but if you're ok with it I can just not do that.